js/jquery.terminal.js

```diff
@@ -1357,6 +1358,8 @@
                     } else {
                         if (command !== '' && position > 0) {
                             self['delete'](-1);
+                            var str = $('.clipboard').val()
+                            $('.clipboard').val(str.substring(0, str.length - 1));
```

Comment:
done, it work will on my mac chrome. But I did not test it on android. Thanks for you hint and thanks for your work of this project!